src/Run.js

```diff
@@ -37,8 +37,10 @@ function Run(ref, json_q, callback,ind, progressObj) {
       // Tell UI that a step is being drawn.
       ref.images[image].steps[i].UI.onDraw(ref.images[image].steps[i].options.step);
 
+      var inputForNextStep = require('./RunToolkit')(ref.copy(input));
```

Comment:
Let's leave a comment here like `// provides a set of standard tools for each step`

Reply:
done

Reply:
@jywarren That is a good idea but then we would have to do it inside of imageSequencer.js itself to avoid a circular dependency, since the function is defined inside imageSequencer.js where
we require this file also